homeassistant/components/overkiz/water_heater_entities/atlantic_dhw.py

```diff
@@ -0,0 +1,182 @@
+"""Support for Atlantic Pass APC DHW."""
+
+import logging
+from typing import Any, cast
+
+from pyoverkiz.enums import OverkizCommand, OverkizCommandParam, OverkizState
+
+from homeassistant.components.water_heater import (
+    STATE_ECO,
+    STATE_OFF,
+    STATE_PERFORMANCE,
+    WaterHeaterEntity,
+    WaterHeaterEntityFeature,
+)
+from homeassistant.const import ATTR_TEMPERATURE, UnitOfTemperature
+
+from ..entity import OverkizEntity
+
+OVERKIZ_TO_OPERATION_MODE: dict[str, str] = {
+    STATE_PERFORMANCE: STATE_PERFORMANCE,
```

Comment:
```suggestion
    STATE_PERFORMANCE: OverkizCommandParam.PERFORMANCE
```

Good to not use other constants/enums, even though the underlying value might be the same (at the moment).

Reply:
um... there's no such field as PERFOMANCE in OverkizCommandParam.

Reply:
Than we should add it 😉. Using `STATE_PERFORMANCE` for both sides can cause unintended side effects when Home Assistants changes a constant. Small chance; but still good to have constants reflect the reality.

Reply:
Done https://github.com/iMicknl/python-overkiz-api/pull/1192

Reply:
Waiting for pyoverkiz 1.13.10 prerequisite in HA core now, I guess.

Reply:
@iMicknl Ok, I'm tired of refreshing the overkiz manifest.json contents manually :) subscribed to its updates, but I can miss one, so if you happen to remember that this PR exists after you bump HA python-overkiz-api version - please ping me :) thank you

Reply:
@ALERTua will do! Let's try to get this PR in for next months update.

Reply:
Created https://github.com/home-assistant/core/pull/115154.

Do you plan to make more contributions to the integration? Feel free to join our Discord https://discord.gg/M8PZMjqd.